railties/test/generators/app_generator_test.rb

```diff
@@ -686,6 +686,19 @@ def test_after_bundle_callback
     end
   end
 
+  def test_dotrails_file_present
+    run_generator
+
+    assert_file ".rails"
```

Comment:
We should also expand the test case to run `bin/rails rails:update` and see that skipped parts are not added again. This way we get full stack coverage.